packages/thirdweb/src/analytics/track/siwe.ts

```diff
@@ -0,0 +1,48 @@
+import type { ThirdwebClient } from "../../client/client.js";
+import { stringify } from "../../utils/json.js";
+import type { Ecosystem } from "../../wallets/in-app/core/wallet/types.js";
+import { track } from "./index.js";
+
+type SiweEvent = {
+  client: ThirdwebClient;
+  ecosystem?: Ecosystem;
+  walletAddress?: string;
+  walletType?: string;
+  chainId?: number;
+  error?: {
+    message: string;
+    code: string;
+  };
+};
+
+/**
+ * @internal
+ */
+export async function trackLogin(event: SiweEvent) {
+  return trackSiweEvent({
+    ...event,
+    action: "login:attempt",
+  });
+}
+
+/**
+ * @internal
+ */
+async function trackSiweEvent(
+  event: SiweEvent & {
+    action: "login:attempt";
```

Comment:
i read this as 'the event before verifying any payload' - but really its the login event, and that can be success of failed if theres an error right?

Reply:
I thought we wanted a single action label error and we'd tell from the error if it succeeded or not?

Reply:
yeah, was just thrown off by the 'attempt' label.

would have just called it `action: login` or `action: authenticate`